main.py

```diff
@@ -154,6 +154,11 @@ def create_circuit(self):
             log.debug('create_circuit result %s %s', result, 409)
             raise Conflict(result)
 
+        if not evc.primary_path and evc.dynamic_backup_path is False:
+            result = "The EVC must have a primary path or allow dynamic paths."
+            log.debug('create_circuit result %s %s', result, 400)
+            raise BadRequest(result)
+
```

Comment:
@ajoaoff I believe this will result in problems for intra-switch EVCs, isn't it?

Reply:
Yes, you're right. I'll fix it.